src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -128,6 +127,8 @@ public DatabaseRecord(string databaseName)
 
         public HashSet<string> UnusedDatabaseIds = new HashSet<string>();
 
+        public List<string> SupportedFeatures = new List<string>();
```

Comment:
shouldn't this be internal?

Reply:
umm.. I don't think so, why?

Reply:
I was thinking about that if we add that as public and in the future we would want to mark it obsolete, if it will be internal, we can easily do it, so it wont be a breaking change